frontend/src/app/specification-form/specification-form.component.html

```diff
@@ -21,14 +21,21 @@ <h4 class="modal-title">Local file upload</h4>
          <p>
            For a GraphQL specification, ApiCenter needs some additional metadata:
          </p>
-         <div *ngFor="let fieldName of objectKeys(additionalFields)">
+         <div *ngFor="let fieldName of objectKeys({
+                                          title: additionalFields.title,
+                                          version: additionalFields.version,
+                                          description: additionalFields.description
+                                          })">
            <input type="text"
                   [placeholder]="fieldName.charAt(0).toUpperCase() + fieldName.slice(1)"
```

Comment:
This has nothing to do with this PR, but I think the description should be a `textbox` while `title` and `version` should be a usual `<input type="text">`.